git/helpers.sh

```diff
@@ -1,3 +1,15 @@
+#
+# Open file or directory in web browser; defaults to current directory and branch
+# Syntax: git-web [$file_or_dir] [$branch]
+#
+function gitweb() {
+  file=${1:-""}
+  git_branch=${2:-$(git symbolic-ref --quiet --short HEAD)}
+  git_project_root=$(git config remote.origin.url | sed -r "s~(\w+@|https://|ssh://(\w+@)?)([^:^/]*)(:[0-9]+|)[:/](.*)\.git~https://\3\4/\5~")
```

Comment:
This line does not work for me. I get:
```
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Reply:
I have replaced this with a simpler and working command here: https://github.com/OtoAnalytics/womply-scripts/pull/33/files#diff-58c49966f0357db4f52a7712cfdc31eee5a934c63088568d737f844cc25dec16R91